dojson/overdo.py

```diff
@@ -55,7 +55,9 @@ def do(self, blob):
 
         for key, value in iteritems(blob):
             for name, creator, field in self._query(key):
-                output[name] = creator(output, key, value)
+                val = creator(output, key, value)
```

Comment:
Another crazy idea that would allow storing `None` values would be to raise e.g. `KeyError`.

### Example
```python
overdo = dojson.Overdo()

@overdo.over('024', '^024..')
def match(self, key, value):
    raise KeyError(key)  # or custom IgnoreKey(key)
```

```diff
+try
-output[name] = creator(output, key, value)
+    output[name] = creator(output, key, value)
+except KeyError:  # or IgnoreKey
+    # logging or pass
```